src/libstore/builtins/buildenv.cc

```diff
@@ -93,8 +93,9 @@ static void createLinks(State & state, const Path & srcDir, const Path & dstDir,
                     auto prevPriority = state.priorities[dstFile];
                     if (prevPriority == priority)
                         throw Error(
-                                "packages '%1%' and '%2%' have the same priority %3%; "
+                                "files '%1%' and '%2%' have the same priority %3%; "
                                 "use 'nix-env --set-flag priority NUMBER INSTALLED_PKGNAME' "
+                                "or 'nix profile install --priority NUMBER INSTALLED_PKGNAME' "
```

Comment:
I’d rather keep this away for now since it isn’t working. Or just specify that you can change the priority _when installing_

Reply:
Done

Reply:
Done